bond/commands/wifi/connect.py

```diff
@@ -0,0 +1,30 @@
+from base64 import b64encode
+
+import bond.proto
+from bond.database import BondDatabase
+
+
+class WifiConnectCommand(object):
+    subcmd = "connect"
+    help = "Connect to WiFi"
+    arguments = {
+        "--ssid": {
+            "help": "The SSID (name) of the WiFi access point to connect to",
+            "required": True,
+        },
+        "--password": {"help": "The password", "required": True},
```

Comment:
Maybe worths saying that the argument should be used in plain text, but the request sends it b64 encoded

Reply:
It's going to be hard to add that while still keeping the message short and clear. 😕

Reply:
I'm going to resolve this but will think of ways of detailing this to our users (via documentation?).